xbmc/addons/GUIViewStateAddonBrowser.cpp

```diff
@@ -40,6 +41,15 @@ CGUIViewStateAddonBrowser::CGUIViewStateAddonBrowser(const CFileItemList& items)
   else
   {
     AddSortMethod(SortByLabel, SortAttributeIgnoreFolders, 551, LABEL_MASKS("%L", "%I", "%L", ""));  // Filename, Size | Foldername, empty
+
+    if (URIUtils::PathStarts(items.GetPath(), "addons://sources/"))
```

Comment:
@Razzeee can you give some more details on how to reproduce this? I'm not sure what the issue is..